components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -235,15 +247,42 @@
   position: absolute;
   right: 0;
   z-index: 10000;
-  min-width: 140px;
-  max-width: 255px;
 }
 .legend-container span {
   font-family: Arial,Verdana,sans-serif;
 }
+.legend-box-controls-open {
+  background: #FFF;
+  height: 20px;
+  border-bottom: 1px solid;
+  border-color: #ccc;
+}
+.legend-box-controls-closed {
+  background: #FFF;
+  height: 25px;
+  cursor: pointer;
+}
+.legend-box-button {
+  float: right;
+  margin-top: 3px;
+  margin-right: 3px;
+  cursor: pointer;
+}
+.legend-box-button:hover {
+  font-weight: bold;
+}
+.legend-hide-show-checkbox {
+  cursor: pointer;
+  float: right;
+  margin-bottom: -1px;
```

Comment:
With this negative margin, on my screen (FF54/Linux, all zoom levels) the chechbox looks ugly:

![image](https://user-images.githubusercontent.com/651980/27303772-d3c19b96-5509-11e7-8ca1-835380e2c9c3.png)

If your intention was to prevent the `border-bottom` from being pushed down, I suggest a negative `margin-top` instead.